app/controllers/sudo_mode_handler.rb

```diff
@@ -12,6 +12,8 @@ class SudoModeHandler
     email: 4,
   }.freeze
 
+  CENTS_THRESHOLD = 500_00
```

Comment:
```suggestion
  THRESHOLD_CENTS = 500_00
```
nitpick: We tend to tack on `_cents` at the end of a variable name as a unit. Unless there's a reason not to, it might be better to use `THRESHOLD_CENTS`

Reply:
makes sense